lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureBlockBlobFileSystem.cs

```diff
@@ -107,7 +110,9 @@ public void CopyToLocal(Uri remoteFileUri, string localName)
         /// </summary>
         public void CopyFromLocal(string localFileName, Uri remoteFileUri)
         {
-            _client.GetBlockBlobReference(remoteFileUri).UploadFromFile(localFileName, FileMode.Open);
+            var blockBlob = _client.GetBlockBlobReference(remoteFileUri);
+            _client.GetContainerReference(blockBlob.Blob.Container.Name).CreateIfNotExists();
+            blockBlob.UploadFromFile(localFileName, FileMode.Open);
```

Comment:
I believe that UploadFromFile also does not exist in the .netcore api and you have to use the Async version.

Reply:
This is just the name of the method. The code inside calls the async method